src/partials/10_header.html

```diff
@@ -18,10 +18,16 @@
         <div class="col text-right">
           <ul>
             <li>
-              <a href="#"><i class="fas fa-user"></i> Login</a>
+              <a class="personal-data" href="#"
+                ><i class="fas fa-user"></i>
+                <p>Login</p></a
+              >
             </li>
             <li>
-              <a href="#"><i class="fas fa-lock"></i> Register</a>
+              <a class="personal-data" href="#"
+                ><i class="fas fa-lock"></i>
+                <p>Register</p></a
+              >
             </li>
```

Comment:
Dlaczego tak dziwnie przestawiłes tutaj `<`?

Reply:
Jak powiem że było/lub samo się zrobiło to pewnie nie uwierzysz ? :D :D No też na to zwróciłem uwagę ale wydawało mi się że podczas kompilowania mi to tak zmieniło :D A może ja przypadkowo kliknąłem... nieważne już poprawiam :D

Reply:
Nie wykluczam, ze edytor cos Ci popsuł ;) No ale na repo takich rzeczy nie ma co trzymać :)